sale_payment_sheet_financial_risk/models/res_partner.py

```diff
@@ -41,6 +41,7 @@ def _get_risk_sale_payment_sheet_domain(self):
         )
         return [
             ("partner_id", "in", commercial_partners.ids),
+            ("invoice_id.payment_state", "!=", "paid"),
```

Comment:
```suggestion
            ("invoice_id.payment_state", "not in", ("in_payment", "paid")),
```

Reply:
The suggested change is not exactly what we are looking for, if it is ‘in_payment’ it should be included because the balance is handled differently, as it is with open invoices for example.
This change is only to cover a rare case where the payment_sheet is left open, but the invoice has already been paid.